src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerController.java

```diff
@@ -50,4 +57,15 @@ public CaseEventHandlerController(List<CaseEventHandler> handlerServices) {
         return noContent().build();
 
     }
+
+    @Operation(description = "Query case event message db to find problematic messages by message type")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK", content = {
+            @Content(mediaType = "application/json", schema = @Schema(implementation = List.class))})
+    })
+    @PostMapping(path = "/messages/jobs/{messageState}", consumes = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Should this be @GetMapping as you are returning some data

Reply:
I thought about it but left is as a post due to the ticket description and AC

Reply:
@mkayad - We should talk about this API a bit since I don't think it was fully detailed.  The LLD mentioned this:  https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1531414005#WA:HandlingCaseEventsInOrderProcessingandDLQs-MessageMonitorJob